examples/smac/feedforward/decentralised/run_ippo_smac_death_masking.py

```diff
@@ -92,10 +94,21 @@ def network_factory(*args: Any, **kwargs: Any) -> Any:
         policy_optimiser=policy_optimiser,
         critic_optimiser=critic_optimiser,
         run_evaluator=True,
-        epoch_batch_size=5,
+        epoch_batch_size=32,
         num_epochs=15,
         num_executors=1,
-        multi_process=True,
+        multi_process=False,
```

Comment:
Why did we disable multi-process?

Reply:
I generally just disable this when I am running experiments locally. I will set it back to True